ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-master.yaml

```diff
@@ -55,15 +55,6 @@ tests:
       IMAGE_REPO: opendatahub-operator
       RELEASE_VERSION: master
     workflow: opendatahub-io-ci-image-mirror
```

Comment:
maybe rename the file with branch name?

Reply:
Maybe we can get rid of the master branch jobs entirely

Reply:
if we do not want "master" i guess in the future it will be "main", so it is ok to stop configs on master.

Reply:
Yeah `master` has the v1 codebase